pkg/cluster/cluster_config.go

```diff
@@ -22,13 +23,68 @@ import (
 // +kubebuilder:rbac:groups="config.openshift.io",resources=ingresses,verbs=get
 
 type Platform string
+
```

Comment:
Unrelated change :( But will not update until other changes needed due to the test instability